R/qenv-eval_code.R

```diff
@@ -44,6 +44,18 @@ setMethod("eval_code", signature = c("qenv", "expression"), function(object, cod
         class = c("qenv.error", "try-error", "simpleError"),
         trace = object@code
       )
+    },
+    warning = function(w) {
```

Comment:
So I think you need to use the withCallingHandlers that chunks used to catch warnings but then continue evaluating the code

Reply:
added.